### PR TITLE
Add labels for chore, discussion and ADR types

### PR DESCRIPTION
**Pull request by jeremyw24**
Add labels "type: chore" and "rfc: discussion"

### CODE REVIEW

File: labels.json [hunk 1]
```diff
@@ -218,5 +218,15 @@
...
     "description": "Issue related to Security",
     "color": "3E83FC"
+  },
+  {
+    "name": "type: chore",
+    "description": null,
+    "color": "006b75"
+  },
+  {
+    "name": "rfc: discussion",
```
Comment (kimburgess) [inline labels.json:228]:
```suggestion
    "name": "type: rfc",
```

Comment (caspiano) [reply labels.json:228]: also `type: discussion`?

Comment (kimburgess) [reply labels.json]: @jeremyw24 to Caspian's point - what's the intent of these? [RFC's](https://en.wikipedia.org/wiki/Request_for_Comments) and discussions are seperate things.

I'll add some support for renaming labels to this automation, but that's not there atm so simpler to get this right first go.

Comment (jeremyw24) [reply labels.json]: @kimburgess there is some human error involved here ;) 
it should be type: discussion == an issue resolved via discussion.
@caspiano requested type: rfc but I messed it up when I created it.

Comment (jeremyw24) [reply labels.json]: added commit with the correction now.

File: labels.json [hunk 2]
```diff
@@ -218,5 +218,20 @@
     "name": "type: security",
     "description": "Issue related to Security",
     "color": "3E83FC"
+  },
+  {
+    "name": "type: chore",
+    "description": null,
```
[the remark this answers is not in the document]
Comment (caspiano) [reply labels.json:224]: I think chores can involve code change, for example, updating due to a breaking change in a library

**Pull request merged**